app/src/main/java/com/appunite/loudius/common/Screen.kt

```diff
@@ -69,5 +70,19 @@ sealed class Screen(val route: String) {
             pullRequestNumber: String,
             submissionDate: String,
         ): String = "reviewers_screen/$pullRequestNumber/$owner/$repo/$submissionDate"
+
+        fun getInitialValues(savedStateHandle: SavedStateHandle) = ReviewersInitialValues(
+            checkNotNull(savedStateHandle[ownerArg]),
```

Comment:
suggestion: id' specify name arguments here, it would be easier to understand if you're passing arguments in the correct order.  Usually it's better to pass named arguments if you're passing more than one argument with the same type.

Reply:
Sounds good for me - added 👌